tvb_build/docker/Dockerfile-multiscale-build

```diff
@@ -0,0 +1,193 @@
+#
+
+FROM neurodebian:buster
+MAINTAINER lia.domide@codemart.ro
+
+ENV DEBIAN_FRONTEND noninteractive
+ENV LANG=C.UTF-8
+
+RUN apt-get update; apt-get install -y automake libtool build-essential git vim octave \
+                       wget python3.7 libpython3.7 libpython3-dev libncurses5-dev libreadline-dev libgsl0-dev cython3 \
+                       python3.7-dev python3-pip python3-numpy python3-scipy python3-matplotlib python3-nose \
+                       ipython3 python3-venv python-virtualenv python3-pyqt5 python-dev libgsl-dev libncurses-dev \
+                       subversion cmake zip unzip; \
+    apt-get clean
+
+RUN wget -q http://www.mpich.org/static/downloads/3.1.4/mpich-3.1.4.tar.gz \
+    && tar xf mpich-3.1.4.tar.gz \
+    && cd mpich-3.1.4 \
+    && ./configure --disable-fortran --enable-fast=all,O3 --prefix=/usr \
+    && make -j$(nproc) \
+    && make install \
+    && ldconfig \
+    && cd .. \
+    && rm -rf mpich-3.1.4 \
+    && rm mpich-3.1.4.tar.gz
+
+# Create user different than root
+RUN useradd -ms /bin/bash docker
+USER docker
+ARG HOME=/home/docker
+RUN mkdir $HOME/env; mkdir $HOME/packages
+
+# Create a dedicated Python env for working in it
+ENV VENV=$HOME/env/neurosci
+RUN python3 -m venv $VENV && python3 -m venv --system-site-packages $VENV; \
+    $VENV/bin/pip3 install --upgrade pip; \
+    $VENV/bin/pip3 install parameters quantities neo "django<1.9" django-tagging future hgapi gitpython sumatra nixio; \
+    $VENV/bin/pip3 install --upgrade nose ipython
+# django<1.9 is needed for pynest - september 2019 - NEST 2.18
+
+# Install from sources libneurosim (NEST boost dependency)
+WORKDIR $HOME/packages
+RUN git clone --depth 1 https://github.com/INCF/libneurosim.git; \
+    cd libneurosim; ./autogen.sh; \
+    mkdir $VENV/build
+WORKDIR $VENV/build
+RUN mkdir libneurosim; \
+    cd libneurosim; \
+    PYTHON=$VENV/bin/python $HOME/packages/libneurosim/configure --prefix=$VENV; \
+    make; make install; ls $VENV/lib $VENV/include; \
+    ln -s /usr/lib/python3.7/config-x86_64-linux-gnu/libpython3.7.so $VENV/lib/
+
+# Install NEST, latest stable version (July 2019)
+ARG NEST_VER=2.18.0
+ARG NEST=nest-simulator-$NEST_VER
+ARG NEST_SRC=$HOME/packages/$NEST
+ARG NEST_INSTALL_DIR=$VENV/nest_build
+ARG NEST_PYTHON_PREFIX=${NEST_INSTALL_DIR}/lib/python3.7/site-packages
+ARG NEST_MODULE_PATH=${NEST_INSTALL_DIR}/lib/nest
+
+WORKDIR $HOME/packages
+RUN wget  https://github.com/nest/nest-simulator/archive/v$NEST_VER.tar.gz -O $HOME/packages/$NEST.tar.gz; \
+    tar xzf $NEST.tar.gz; rm $NEST.tar.gz; \
+    cd $NEST; \
+    cmake -DCMAKE_INSTALL_PREFIX=${NEST_INSTALL_DIR} \
+#          -Dwith-mpi=ON  \
+          -Dwith-python=3 \
+          ###-Dwith-music=ON \
+          -Dwith-libneurosim=$VENV \
+          -DPYTHON_LIBRARY=/usr/lib \
+          -DPYTHON_INCLUDE_DIR=/usr/include/python3.7m \
+          ${NEST_SRC}; \
+    make; make install
+
+# Install nestml
+RUN $VENV/bin/pip3 install setuptools; \
```

Comment:
this line fails with the error
```
error: numpy 1.16.2 is installed but numpy>=1.17 is required by {'astropy'}
```

Reply:
I have removed completely this Dockerfile. It shouldn't have been there in the first place.